src/types/redis_hash.cc

```diff
@@ -355,4 +383,36 @@ rocksdb::Status Hash::Scan(const Slice &user_key, const std::string &cursor, uin
   return SubKeyScanner::Scan(kRedisHash, user_key, cursor, limit, field_prefix, fields, values);
 }
 
+Status Hash::ParseRangeLexSpec(const std::string &min, const std::string &max, HashRangeSpec *spec) {
```

Comment:
We can unite these same structure and methods of HashRangeSpec and ZRangeSpec next.

Reply:
why not this pr, i don't think we should import this function

Reply:
I have no strong opinion for this. Could you merge these structures and functions in this PR? @tanruixiang

Reply:
> I have no strong opinion for this. Could you merge these structures and functions in this PR? @tanruixiang

Ok.